special-interest-groups/sig-ddl/README.md

```diff
@@ -5,6 +5,14 @@ DDL SIG covers the TiDB DDL and parser. Our work includes
 * Fixing all kinds of DDL-related and parser-related bugs.
 * Improving the performance.
 
+## Learning Materials
+
+* Basic Theory Related Articles
+    * DDL Related Articles: [Online, Asynchronous Schema Change in F1](https://static.googleusercontent.com/media/research.google.com/zh-CN//pubs/archive/41376.pdf), [Deep analysis of Google F1's schema change algorithm(CN)](https://github.com/ngaut/builddatabase/blob/master/f1/schema-change.md)
+    * Parser Related Articles: [Bison manual](https://www.gnu.org/software/bison/manual/html_node/index.html), [Compilers: Principles, Techniques, and Tools](https://www.amazon.com/Compilers-Principles-Techniques-Tools-2nd/dp/0321486811v)
+* Architecture Related Articles: [TiDB DDL architecture](https://github.com/pingcap/tidb/blob/master/docs/design/2018-10-08-online-DDL.md)
+* Source Code Implementation Related Articles: based on the v1.1 [source-code](https://github.com/pingcap/tidb/tree/source-code) version of the [DDL source code reading(CN)](https://pingcap.com/blog-cn/tidb-source-code-reading-17/) and [Parser implementation(CN)](https://pingcap.com/blog-cn/tidb-source-code-reading-5/)
+
```

Comment:
```suggestion

## Contributing

Please feel free to explore our [GitHub project](https://github.com/pingcap/tidb/projects/40).

```

How about post our GitHub project here?

Reply:
Of course. But I use `https://github.com/orgs/pingcap/projects/28` instead of `https://github.com/pingcap/tidb/projects/40`.